src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+:root {
+  --current-text: #000;
+  --brend-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  font-family: "Roboto", "Arial", sans-serif;
+  font-size: 12px;
```

Comment:
better to move these styles to html selector

usually, we just reset some default styles in the body

Reply:
i will fix it